app/controllers/agent/DeductionsSummaryController.scala

```diff
@@ -57,9 +57,13 @@ class DeductionsSummaryController @Inject()(deductionBreakdown: DeductionBreakdo
 							val viewModel = AllowancesAndDeductionsViewModel(liabilityCalc.calculation)
 							auditingService.extendedAudit(AllowanceAndDeductionsResponseAuditModelNew(getMtdItUserWithNino(), viewModel))
 							Ok(deductionBreakdownViewNew(viewModel, taxYear, backUrl(taxYear), isAgent = true))
+						case error: LiabilityCalculationError if error.status == NOT_FOUND =>
+							Logger("application").error(
+								s"[Agent][DeductionsSummaryController][showDeductionsSummary[$taxYear]] No deductions data found.")
```

Comment:
Should this be:
Logger("application").info(...

Reply:
Yes changed it now.